doc/book/admin/server_introspection.rst

```diff
@@ -169,6 +169,49 @@ But, to force Tarantool to release memory, you can
 call :doc:`box.snapshot() </reference/reference_lua/box_snapshot>`, stop the server instance,
 and restart it.
 
+.. _admin-inspect_traffic:
+
+--------------------------------------------------------------------------------
+Inspect traffic
+--------------------------------------------------------------------------------
+
+Inspecting the binary traffic is the boring task. We offer a
+`Wireshark plugin <https://github.com/tarantool/tarantool-dissector>`_ to
+simplify analysis of Tarantool's traffic.
+
+Please, consider several simple steps to enable the plugin.
+
+Clone the tarantool-dissector repository:
+
+.. code-block:: console
+
+    git clone https://github.com/tarantool/tarantool-dissector.git
+
+Copy or symlink the plugin files into the Wireshark plugin directory:
+
+.. code-block:: console
+
+    mkdir -p ~/.local/lib/wireshark/plugins
+    cd ~/.local/lib/wireshark/plugins
+    ln -s /path/to/tarantool-dissector/MessagePack.lua ./
+    ln -s /path/to/tarantool-dissector/tarantool.dissector.lua ./
+
+(Please, refer to chapter
+`Plugin folders <https://www.wireshark.org/docs/wsug_html_chunked/ChPluginFolders.html>`_
+in the Wireshark documentation for location of the plugin directory on macOS and
+Windows.)
+
+Run Wireshark GUI and ensure that the plugins are loaded:
+
+* Open ``Help`` -> ``About Wireshark`` -> ``Plugins``.
```

Comment:
```suggestion
* Open :guilabel:`Help` > :guilabel:`About Wireshark` > :guilabel:`Plugins`.
```
Consider using the `:guilabel:` directive to indicate GUI elements, as mentioned in our [style guide](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/gui/).

Reply:
Missed it, thanks! Updated.